roles/apigw/templates/apigw_routes_config_map.yaml.j2

```diff
@@ -277,5 +277,18 @@ data:
         }
         PAYLOAD
         authConfig "cron"
-        
+
+        # Configure a kafka-ws service
+        (curl -o /dev/null ${kong}/apis -sS -X POST \
+            --header "Content-Type: application/json" \
+            -d @- ) <<PAYLOAD
+        {
+            "name": "kafka-ws",
+            "uris": "/kafka-ws",
```

Comment:
The URI changed in @thiagodpf newer modifications.

Reply:
The idea is...

Request for API Gateway:
`http://localhost:8000/kafka-ws/*`

Redirects to service:
`http://kafkaws:5000/api/*`

Would have to activate the JWT verification plugins on the API Gateway for the route:
`http://kafkaws:5000/api/v1/ticket`

Route `http://kafkaws:5000/api/v1/topics` cannot be intercepted by the plugins since the request no longer contains the JWT in the header.